src/test/java/org/jabref/logic/importer/fetcher/SpringerFetcherTest.java

```diff
@@ -132,6 +134,20 @@ public void supportsBooleanANDSearch() throws Exception {
                           .map(Optional::get).forEach(authorField -> assertTrue(authorField.contains("Redmiles")));
     }
 
+    /**
+     * Ensure that different page return different entries
+     */
+    @Test
+    public void pageSearchReturnsUniqueResultsPerPage() throws Exception {
```

Comment:
That is a good general test that should exist for every paged fetcher, thus I suggest to move it to the general test class (not sure about the name, maybe `PagedSearchFetcherTests`?)